I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/chapter01.rpy", line 87, in script
    "You walk down the hallway until you find an unoccupied room. Pushing open the door with some effort."
OSError: Couldn't find file ''.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/chapter01.rpy", line 87, in script
    "You walk down the hallway until you find an unoccupied room. Pushing open the door with some effort."
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\ast.py", line 736, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\exports.py", line 1419, in say
    who(what, *args, **kwargs)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\character.py", line 1239, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\character.py", line 900, in do_display
    display_say(who,
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\character.py", line 647, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\display\core.py", line 3336, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\display\core.py", line 4036, in interact_core
    renpy.audio.audio.periodic()
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\audio\audio.py", line 1160, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/nightly-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\audio\audio.py", line 1138, in periodic_thread_main
    periodic_pass()
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\audio\audio.py", line 1080, in periodic_pass
    c.periodic()
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\audio\audio.py", line 517, in periodic
    topf = load(filename)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\audio\audio.py", line 69, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\Ethan\Downloads\renpy-7.4.11-sdk\renpy\loader.py", line 804, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file ''.

Windows-10-10.0.19042
Ren'Py 8.0.0.22011702n
donor 1.0
Tue Jan 25 15:46:34 2022
